src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -79,6 +97,24 @@ std::shared_ptr<ov::Model> create_dummy_model(const IONodeDescriptorMap& paramet
     return std::make_shared<ov::Model>(results, parameters);
 }
 
+void set_batch_config(std::shared_ptr<NPUBackends> backends, Config& config) {
```

Comment:
No need to pass `backends` as a param, you just pass a boolean. It would make the interface clearer

Reply:
done, thanks!